platform/broadcom/sonic-platform-modules-accton/as7726-32x/classes/thermalutil.py

```diff
@@ -116,16 +113,10 @@ def get_thermal_temp(self):
 
 def main():
     thermal = ThermalUtil()
-    print "termal1=%d" %thermal._get_thermal_val(1)
-    print "termal2=%d" %thermal._get_thermal_val(2)
-    print "termal3=%d" %thermal._get_thermal_val(3)
-    print "termal4=%d" %thermal._get_thermal_val(4)
-    print "termal5=%d" %thermal._get_thermal_val(5)    
-#
-#    print 'get_size_node_map : %d' % thermal.get_size_node_map()
-#    print 'get_size_path_map : %d' % thermal.get_size_path_map()
-#    for x in range(thermal.get_idx_thermal_start(), thermal.get_num_thermals()+1):
-#        print thermal.get_thermal_to_device_path(x)
-#
+    logging.debug('termal1=%d', thermal._get_thermal_val(1))
+    logging.debug('termal2=%d', thermal._get_thermal_val(2))
+    logging.debug('termal3=%d', thermal._get_thermal_val(3))
+    logging.debug('termal4=%d', thermal._get_thermal_val(4))
+    logging.debug('termal5=%d', thermal._get_thermal_val(5))
```

Comment:
Spelling: s/termal/thermal

Reply:
OK. I fixed it in 2nd commit. Please check it.